frontend/simple/views/GroupDashboard.vue

```diff
@@ -41,6 +41,14 @@
         <dashboard-section title="Group Settings">
           <group-settings :group="currentGroupState" />
         </dashboard-section>
+
+        <modal-proposal-mincome></modal-proposal-mincome>
+        <!--  TODO: on next MR with logic for each step -->
+        <!-- <modal-proposal-add-member></modal-proposal-add-member> -->
+        <!-- <modal-proposal-remove-member></modal-proposal-remove-member> -->
+        <!-- <modal-proposal-rule-change-rule></modal-proposal-rule-change-rule> -->
+        <modal-proposal-rule-add-member></modal-proposal-rule-add-member>
+        <!-- <modal-proposal-rule-remove-member></modal-proposal-rule-remove-member> -->
```

Comment:
I'm unclear what's going on here. The idea was to have a single component here representing a modal. So I think somewhere there was a misunderstanding. It would be a generic modal, and *any .vue component* could be swapped into it via, for example, a `<component :is="binding">` type field. See the Vue.js documentation for how to do that.

So there should be no `ModalForm.vue` file, and there shouldn't exist `ModalProposalMincome` or ModalProposalRuleAddMember`, at least they shouldn't have "Modal" in the name, and they shouldn't be listed here.

Here there should be just the generic modal component that listens for events and based on those events swaps out its innards for a different .vue file.